release: v1.49.0 (#5418)
apollographql/router: .changesets/maint_bryn_disable_tests_apollo_key_missing.md

> ### Disable GraphOS tests when apollo key not present ([PR #5362](https://github.com/apollographql/router/pull/5362))
> 
> A number of tests require APOLLO_KEY and APOLLO_GRAPH_REF to be present to execute successfully.
> These are now skipped if these env variables are not present.

Reviewer: ```suggestion
### Skip GraphOS tests when Apollo key not present ([PR #5362](https://github.com/apollographql/router/pull/5362))

Some tests require `APOLLO_KEY` and `APOLLO_GRAPH_REF` to execute successfully.
These are now skipped if these env variables are not present.
```
